spec/system/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -473,6 +473,123 @@
       login_publisher(publisher: forbidden_publisher)
       visit publishers_jobseeker_profiles_path
       expect(page).not_to have_content(profile.full_name)
+
+      click_on I18n.t("nav.sign_out")
+    end
+
+    context "if the jobseeker is already hidden from the school" do
+      before do
+        profile.excluded_organisations << forbidden_organisation
+      end
+
+      it "does not allow the jobseeker to hide themselves from the school again" do
+        visit jobseekers_profile_path
+        click_on I18n.t("jobseekers.profiles.show.set_up_profile_visibility")
+        choose "Yes", visible: false
+        click_on I18n.t("buttons.save_and_continue")
+
+        field = find_field("Name of school or trust")
+        field.fill_in(with: forbidden_organisation.name)
+        field.native.send_keys(:tab)
```

Comment:
What's the purpose of this in this test?

Reply:
It triggers the JS autocomplete search thing, to actually select the suggested result.